doc/source/getting_started/running_mechanical.rst

```diff
@@ -182,29 +185,13 @@ You can then use this output to debug why Mechanical isn't launching.
 Debug from the command line
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 In some cases, debugging why Mechanical isn't launching might require
```

Comment:
@sbradrick Here, ``launch`` would be in double back ticks as it is referring to code. And, we always replace "in order to" with "to" to avoid the unnecessary words.